scripts/dartagnan

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env sh
+#
+# dartagnan -- script to simplify running dartagnan from the commandline
+#
+# Options:
+# * -cat      : memory model
+# * -target   : target architecture
+# * -property : property to be checked
+#
+# Environment variables:
+#
+# * DAT3M_JAVA : path to the java runtime binary to be used
+# * DAT3M_HOME : path to dat3m installation directory
+set -e
+
+java_cmd="${DAT3M_JAVA}"
+jar=${DAT3M_HOME}/dartagnan/target/dartagnan.jar
+args=""
+this="$0"
+
+if [ -z "${java_cmd}" ]; then
+    java_cmd="java"
+fi
+
+usage () {
+    echo "${this} [-cat MODEL] [-target TARGET] [-property PROPERTY] <FILE>"
+}
+
+while [ "$#" != "0" ]; do
+    case "$1" in
+        -cat)
+            shift
+            cat="$1"
+        ;;
+        -target)
+            shift
+            target="$1"
+        ;;
+        -property)
+            shift
+            property="$1"
+        ;;
+        -help)
+            usage
+            exit
+        ;;
+        *)
+            args="${args} $1"
+        ;;
+    esac
+    shift
+done
+
+# cat argument is mandatory. If the argument is not a file, we guess it to be
+# a file in DAT3M_HOME/cat with .cat suffix.
+if [ -z "${cat}" ]; then
+    echo "error: no cat file specified"
+    exit 1
+elif [ ! -f "${cat}" ]; then
+    cat="${DAT3M_HOME}/cat/$cat".cat
+fi
+
+# if target is not given, we try to guess the target from the cat file
+if [ -z "${target}" ]; then
+	case "$cat" in
+		*aarch64*)
+        		target=arm8
+                ;;
+		*c11*)
+        		target=c11
+                ;;
+		*imm*)
+        		target=imm
+                ;;
+		*lkmm*)
+        		target=lkmm
```

Comment:
```suggestion
        		target=linux-kernel
```

Reply:
This one is still wrong

Reply:
I think Hernán marked the wrong part. The target is correct, but the cat model is `linux-kernel.cat` and not `lkmm.cat`, so the cat-file won't be detected correctly.

Reply:
Yes, I was referring to the cat file, i.e. the `lkmm` at line 75

Reply:
ok, I see it now.